### PR TITLE
Support feedparser 6.0

### PR DESCRIPTION
**Pull request by lazka**
This removes the pre-parse check that was using lots of private API
that is now gone. Ideally this should be implemented using urllib
if it's still required.

### CODE REVIEW

**Comment by lazka** [comment]
@declension

File: pyproject.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ packages = [
 [tool.poetry.dependencies]
 python = "^3.7"
 mutagen = "^1.44"
-feedparser = "^5.2"
+feedparser = "^5.2 || ^6.0"
```
Comment (declension) [inline pyproject.toml:21]: woah, you can do that? What does it mean - choose whichever works?

**Review by declension — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (lazka) [reply pyproject.toml:21]: good question, there is only one case documented for specifying the Python version: https://python-poetry.org/docs/dependency-specification/#multiple-constraints-dependencies

It means either of those constrains with npm/composer, and poetry seems to understand it as well.

Comment (declension) [reply pyproject.toml:21]: TIL, thanks :)

Comment (lazka) [reply pyproject.toml:21]: Just checked: When you create a dist with `poetry build` it converts it to `feedparser>=5.2,<7.0` in the generated setup.py.

**Pull request merged**